spring-xd-dirt/src/main/java/org/springframework/xd/dirt/plugins/AbstractMessageBusBinderPlugin.java

```diff
@@ -122,8 +130,14 @@ private void startTapListener(CuratorFramework client) {
 	 *
 	 * @param module the module whose consumer and producers to bind to the {@link MessageBus}.
 	 */
-	protected final void bindConsumerAndProducers(Module module) {
+	protected final void bindConsumerAndProducers(final Module module) {
+		String xdHistory = module.getDeploymentProperties() != null
+				? module.getDeploymentProperties().get(BusProperties.TRACK_HISTORY)
+				: null;
+		boolean trackHistory = Boolean.valueOf(xdHistory);
 		Properties[] properties = extractConsumerProducerProperties(module);
+		Map<String, Object> historyProperties = extractHistoryProperties(module);
```

Comment:
Yes it can; I thought about it while doing my final review but I didn't feel it worth the effort given that it's only done on deployment and not at runtime. I'm happy to change it though.